plugins/main/vlan/vlan.go

```diff
@@ -53,10 +53,10 @@ func loadConf(bytes []byte) (*NetConf, string, error) {
 		return nil, "", fmt.Errorf("failed to load netconf: %v", err)
 	}
 	if n.Master == "" {
-		return nil, "", fmt.Errorf(`"master" field is required. It specifies the host interface name to create the VLAN for.`)
+		return nil, "", fmt.Errorf("\"master\" field is required. It specifies the host interface name to create the VLAN for.")
```

Comment:
This was the precise example for backtick usage, but ok, lets keep everywhere `"` :)

Reply:
Yeap, I also read some docs about double quotes and back quote... It says that back quote is usually used for multi lines, html and regular expression. So I think both are OK here.